socorro/unittest/processor/__init__.py

```diff
@@ -21,3 +24,16 @@ def create_basic_fake_processor():
     fake_processor.config.logger = logging.getLogger(__name__)
     fake_processor.processor_notes = []
     return fake_processor
+
+
+def get_basic_config():
+    config = CDotDict()
+    config.logger = Mock()
+    return config
+
+
+def get_basic_processor_meta():
+    processor_meta = DotDict()
+    processor_meta.processor_notes = []
+    processor_meta.quit_check = lambda: False
+    return processor_meta
```

Comment:
I extracted these from the processor tests where they were written in like 20 times. Seemed silly. I wanted to turn them into pytest fixtures, but since we're still using unittest scaffolding for those tests, that wasn't possible.